src/components/button/Button.ts

```diff
@@ -0,0 +1,12 @@
+import { render } from '../../utils/domHandlers';
+
+export const Button = (label: string, onClick: () => void, className?: string, parentId?: string): HTMLElement => {
+  const button = document.createElement('div');
+  button.setAttribute('id', 'button');
```

Comment:
ustawmy mu data id np `button-${className}` i wtedy przetestujmy. Id button do buttona nie jest eleganckie

Reply:
Zmienna className nie jest wymaganym parametrem, więc nie jestem przekonana czy tak powinnam to nazwać..

Reply:
W takim razie, potrzebujemy innej nazwy na id

Reply:
ok poprawiłam